docs/assets/js/bibtex-unicorn.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Edit to the original bibtex_js_draw function
+ * Bibtex files need to be entered into individual <textarea>'s with class "reference"
+ * Generated content replaces the inputted bibtex data in each <textarea>
+ * Allows user to write notes/subheadings in between references
+ */
+$( document ).ready( function bibtex_js_draw_unicorn() {
+  var refs = document.querySelectorAll(".reference");
+  refs.forEach(ref => {
+    (new BibtexDisplay()).displayBibtex($(ref).val(), $("#bibtex_display") );
```

Comment:
Can we reuse one `BibtexDisplay` instance for all references or does it have some internal persistence issues?